tests/php/Authenticators/LDAPChangePasswordHandlerTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SilverStripe\LDAP\Tests\Authenticators;
+
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Control\Session;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Dev\SapphireTest;
+use SIlverStripe\LDAP\Authenticators\LDAPAuthenticator;
+use SIlverStripe\LDAP\Authenticators\LDAPChangePasswordHandler;
+use SIlverStripe\LDAP\Forms\LDAPChangePasswordForm;
```

Comment:
That should be use SilverStripe\LDAP\Forms\LDAPChangePasswordForm; (with a lowercase i)

Reply:
https://github.com/silverstripe/silverstripe-ldap/issues/8